src/flask_httpauth.py

```diff
@@ -195,6 +195,49 @@ def ensure_sync(self, f):
             return f
 
 
+class HTTPCookieAuth(HTTPAuth):
+    def __init__(self, scheme=None, realm=None, cookie_name=None):
```

Comment:
Scheme and Realm apply to HTTP Authentication. I don't see how they help when using cookies.